examples/ecommerce/src/App.vue

```diff
@@ -210,6 +180,32 @@
               </template>
             </ais-panel>
           </div>
+
+          <footer class="container-filters-footer" data-layout="mobile">
+            <ais-clear-refinements class="container-filters-footer-button-wrapper">
```

Comment:
This button should:

- [x] Reset the filters
- [ ] Close the overlay
- [ ] Have the same design when not disabled than when disabled (because it's still clickable to close the overlay)
- [ ] Be renamed to "Reset filters"
- [ ] Not contain the image

Reply:
6f1ed6c